### PR TITLE
Add content on performance benchmarks.

### PR DESCRIPTION
**Pull request by khushi-411**
This PR adds content on performance. Follows issue: https://github.com/numpy/numpy.org/issues/370

Please have a look. Thanks!

Rendered markdown link (for ease of readability): https://github.com/khushi-411/numpy.org/blob/master/content/en/benchmark.md

Future work which should be added here according to me are:

* [ ] **Algorithms:** Classification of Algorithms. Here, I classified it into 5 categories (Transonic jit, Transonic boost, Numba, NumPy, Pure NumPy). Future aims will be to update this into Cython, Pythran, Numba, TransonicJit, Transonic boost, Transonic Cython is any other relevant algorithms that I'm presently not sure about.
* [ ] **Graphs:** Do we need to iterate over different datasets? We can just select any one or two datasets and the number of times those algorithms are to be implemented and then plot the graph. Does it sound viable?
* [ ] **Content:** Improve Documentation (Include introduction). I mentioned various libraries in the post. Is that needed? I think that it can be presented in a better way. What do you think about how can I improve it?
* [ ] **Code Quality:** Profile code and optimize it.

cc: @mattip

### CODE REVIEW

File: content/en/benchmark.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  <td>Pure NumPy</td>
+  <td><a href = "https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/bench_numpy_highlevel.py">bench_pure_numpy_highlevel.py</a></td>
+</tr>
+  </table>
+</html>
+
+
+## Conclusion
+
+* These benchmarks are run with an Intel Core i7 processor.
```
Comment (rgommers) [inline content/en/benchmark.md:122]: This probably belongs in the caption of the graph. With a bit more details about the exact CPU. For example:
```
$ cat /proc/cpuinfo | grep Intel
```
And then slightly reformat like: `Intel i9-7920X @ 2.90GHz`

File: content/en/benchmark.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  <td>169.4</td>
+ </tr>
+</table>
+</html>
+
+## Code Folders
+
+* These codes are originally taken from [here](https://github.com/paugier/nbabel)
+* Benchmark code: [here](https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/benchmark-2.py)
+* Graph: [here](https://github.com/khushi-411/numpy-benchmarks/blob/0.3/python/plot-modified-1.py)
```
Comment (rgommers) [inline content/en/benchmark.md:88]: Linking to your own repo is fine for now. We can make it a separate NumPy repo or put the scripts in an existing repo somewhere, when you are closer to done with this PR.

File: content/en/benchmark.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+![output-modify-2-new](https://user-images.githubusercontent.com/62256509/127653679-5151e22d-d141-4621-abb9-b3e6dd7988b7.jpg)
+
+
+## Implemented Algorithms
+
+Till this point 5 algorithms are implemented as mentioned bellow:
+
+### NumPy
+
+NumPy is a most fundamental package for scientific computing in Python. The major advantage of using NumPy is these gives same computational speed as C and Fortran. Let's see our inplementation details:
```
Comment (rgommers) [inline content/en/benchmark.md:13]: I'm thinking what should come next is indeed a comparison with C/C++ and Fortran, like this sentence says?

The accelerators can be a second part of the story.

File: content/en/benchmark.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+# NumPy Benchmarking
+
+
+![output-modify-2-new](https://user-images.githubusercontent.com/62256509/127653679-5151e22d-d141-4621-abb9-b3e6dd7988b7.jpg)
```
Comment (rgommers) [inline content/en/benchmark.md:4]: It would be better to commit the figure in this PR. The place to put it is `static/images/content_images/`

**Review by rgommers — COMMENTED** [review]
Thanks for working on this @khushi-411! Looks like a good start.

File: content/en/benchmark.md (hunk 3 above)
Comment (mattip) [reply content/en/benchmark.md:13]: The text should explain that this is a numerical simulation of the N-body problem and giv ethe constraints of the problem

C/C++ and Fortran code simulations can be found in the original repo https://github.com/paugier/nbabel/

File: content/en/benchmark.md (hunk 1 above)
Comment (khushi-411) [reply content/en/benchmark.md:122]: Apologize, I will update that. Thanks!

File: content/en/benchmark.md (hunk 2 above)
Comment (khushi-411) [reply content/en/benchmark.md:88]: Sure definitely, these are just sample codes. I feel that these need to be improved, and once it's finalized, we can create a repo or update somewhere in the existing repo as you'll wish. Thanks!

**Pull request closed without merging**